sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureStorageFileAttributeView.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.nio;
+
+import java.nio.file.attribute.FileAttributeView;
+
+/**
+ * Provides support for properties specific to Azure Blob Storage such as tier.
+ *
+ * {@inheritDoc}
+ */
+public class AzureStorageFileAttributeView implements FileAttributeView {
+
+    /**
+     * Returns {@code "azureStorage"}
```

Comment:
why is the string for this azureStorage and not AzureStorage

Reply:
just asking cause we call containers AzureStorageBlobContainer

Reply:
It seems the names of other attribute views begins with a lower case letter